tests/integration/test_bincount.py

```diff
@@ -105,6 +161,23 @@ def test_bincount_weights_high_bins(dtype):
     assert allclose(out_np, out_num)
 
 
+@pytest.mark.parametrize(
+    "weights",
+    [
+        None,
+        pytest.param(np.array((0.5,)), marks=pytest.mark.xfail),
+        pytest.param(np.array((22,)), marks=pytest.mark.xfail),
+    ],
+    ids=str,
+)
+def test_bincount_size_one(weights):
+    arr_np = np.random.randint(0, 255, size=1)
+    arr_num = num.array(arr_np)
+    bins_np = np.bincount(arr_np, weights=weights)
+    bins_num = num.bincount(arr_num, weights=weights)
+    assert np.array_equal(bins_np, bins_num)
```

Comment:
Can you elaborate on the failures here, e.g. there are actual differing value from the libraries?